workflowsteps.md

```diff
@@ -0,0 +1,31 @@
+# Workflow Steps Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+A workflow step represents a single step in the reviewing process, e.g. the accept/reject step.  
+
+All endpoints mentioned here require authentication, but no specific permissions.
+
+## Main Endpoint
+**/api/config/workflowsteps**   
+
+Provide access to the configured workflow steps. It returns the list of configured workflow-steps.
+
+## Single Workflow Step Definition
+**/api/config/workflowsteps/<:step-name>**
+
+Provide detailed information about a specific workflow step. An example JSON response document:
+```json
+{
+  	"id": "editstep",
+  	"actions": [
+  	    "approve",
+  	    "reject",
+  	    "edit_metadata"
+  	],
+  	"type": "workflowstep"
+}
+```
+
+The **actions** property contains the list of actions the user is authorized to perform in this step:
+* The **edit_metadata** action implies the user can use the PATCH on the workflow item's submission sections to edit the metadata.
+* Other actions are considered to be actions sent to REST using a POST to the claimed task
```

Comment:
Based on my previous question, if the actions are configurable on the backend, then how do we know which actions use POST vs PATCH?  Should they simply *all* use POST as implied by the REST Contract here: https://github.com/DSpace/Rest7Contract/blob/master/claimedtasks.md#post-method-single-resource-level?  It seems like it might be complex to determine which can support the PATCH method.

Reply:
The `edit_metadata` is a separate use case explained here, which doesn't use https://github.com/DSpace/Rest7Contract/blob/master/claimedtasks.md#post-method-single-resource-level

I think all other actions can be supported using the POST from https://github.com/DSpace/Rest7Contract/blob/master/claimedtasks.md#post-method-single-resource-level